src/test/java/teammates/test/cases/ui/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -35,11 +36,25 @@ public static void classSetup() {
     
     @Test
     public void testAll() throws Exception {
+        testModerationHint();
         testEditResponse();
         testAddResponse();
         testDeleteResponse();
     }
     
+    private void testModerationHint() throws Exception {
+        ______TS("verify moderation hint");
+        
+        submitPage = loginToInstructorEditStudentFeedbackPage("IESFPTCourseinstr", "student1InIESFPTCourse@gmail.tmt",
+                "session1InIESFPTCourse");
+        
+        submitPage.verifyHtmlPart(By.className("navbar"), "/instructorEditStudentFeedbackHintLess.html");
+        submitPage.click(By.id("moderationHintButton"));
```

Comment:
`FeedbackSubmitPage.java` also used by `StudentFeedbackSubmitPageUiTest.java`.
I am thinking that since the `moderationHintButton` is only used by the Instructor's Moderation page, it may not be appropriate for `FeedbackSubmitPage.java .java` to hold the method `clickModerationHintButton()`. May put in the `InstructorEditStudentFeedbackPageUiTest` instead